tests/db/test_bigchain_api.py

```diff
@@ -120,8 +120,22 @@ def test_read_transaction(self, b, user_vk, user_sk):
         block = b.create_block([tx_signed])
         b.write_block(block, durability='hard')
 
-        response = b.get_transaction(tx_signed["id"])
+        response, status = b.get_transaction(tx_signed["id"], include_status=True)
+        # add validity information, which will be returned
+        assert util.serialize(tx_signed) == util.serialize(response)
+        assert status == 'undecided'
```

Comment:
Better to use variable names if we decide to change the strings later.
`assert status == b.TX_UNDECIDED`